src/utils.ts

```diff
@@ -61,7 +61,7 @@ function createErrorMiddleware(
       if (!error) {
         return done();
       }
-      log.error(`MetaMask - RPC Error: ${error.message}`, error);
+      log.debug(`MetaMask - RPC Error: ${error.message}`, error);
```

Comment:
Would `log.warn` do the trick? Would keep it on `stderr` for non-browser environment.

```suggestion
      log.warn(`MetaMask - RPC Error: ${error.message}`, error);
```

Reply:
I think this is the most conservative change that will solve the reported issue. Alright, let's go with it 👍